### ISSUE TITLE
[Bug] Game refuses to load with 1.2.0b

### ISSUE THREAD

**Issue by brisingraerowing** [SuperMartijn642/ChunkLoaders#59]
**Version Info**
- Minecraft: 1.18.2
- Chunk Loaders: 1.2.0b
- Forge: 40.1.60

**Are you using OptiFine?:** No

**Description of the Bug**
Game refuses to load

**Steps to Reproduce**
Load game with 1.2.0b

**Screenshots**
N/A

**Log**

[Game Output](https://paste.ee/p/wgfvn)

Relevant Lines:
```text
[16:34:19] [main/ERROR] [ne.mi.fm.lo.LanguageLoadingProvider/LOADING]: Missing language javafml version [40.1.58,) wanted by chunkloaders-1.2.0b-forge-mc1.18.jar, found 40
```

**Comment by vizthex123** [comment]
same issue here, this is my log: https://gist.github.com/vizthex123/3ee40ca128ae56724955be19d0a69a97

**Comment by SuperMartijn642** [comment]
Fixed now in Chunk Loaders 1.2.0c for 1.18.

Thank you for reporting the issue!
